WordPress/Classes/Utility/Ratings/AppRatingUtility.m

```diff
@@ -65,10 +65,25 @@ + (void)checkIfAppReviewPromptsHaveBeenDisabled:(AppRatingCompletionBlock)succes
 {
     NSURL *url = [NSURL URLWithString:AppReviewPromptDisabledUrl];
     NSURLRequest *request = [[NSURLRequest alloc] initWithURL:url];
-    AFHTTPRequestOperation* operation = [[AFHTTPRequestOperation alloc] initWithRequest:request];
-    operation.responseSerializer = [[AFJSONResponseSerializer alloc] init];
-    [operation setCompletionBlockWithSuccess:^(AFHTTPRequestOperation *operation, id responseObject)
-    {
+    NSURLSession *session = [NSURLSession sessionWithConfiguration:[NSURLSessionConfiguration ephemeralSessionConfiguration]];
+    NSURLSessionDataTask *task = [session dataTaskWithRequest:request completionHandler:^(NSData *data, NSURLResponse *response, NSError *error) {
+        if (error || data == nil) {
+            // Let's be optimistic and turn off throttling by default if this call doesn't work
+            [self resetReviewPromptDisabledStatus];
+            if (failure) {
+                failure();
+            }
+            return;
+        }
+        id responseObject = [NSJSONSerialization JSONObjectWithData:data options:nil error:nil];
```

Comment:
If you see the documentation of the method, the method will return nil if there is an error, and because we are not doing anything with the error I just check for nil on the response.